Misc/NEWS.d/next/Library/2018-10-04-17-23-43.bpo-34898.Wo2PoJ.rst

```diff
@@ -0,0 +1 @@
+Add `mtime` argument to `gzip.compress` for reproducible output.
```

Comment:
Add "Patch by ..."

Add also a What New's entry. Add your name in Misc/ACKS if it was not added yet.